Package.swift

```diff
@@ -20,5 +20,5 @@ let package = Package(
       dependencies: ["Lottie", "SnapshotTesting"],
       path: "Tests",
       exclude: ["Artifacts"],
-      resources: [.process("Samples"), .process("__Snapshots__")]),
+      resources: [.copy("Samples"), .process("__Snapshots__")]),
```

Comment:
I updated the snapshot tests to include the full path of the sample json file in the snapshot name.

This was necessary to `.gitignore` snapshot of samples in the `Samples/Private` folder, and also removes the potential for naming conflicts that could happen previously.
 
 To do this, we have to change the resource strategy from `process` (which takes all of the resource files and just includes them directly in the bundle's root folder and doesn't preserve any subdirectory information) to `copy` (which copies the exact directory structure).